src/sagas/wallet.js

```diff
@@ -537,11 +534,7 @@ export function* handleNewTx(action) {
   // Set the notification click, in case we have sent one
   if (notification !== undefined) {
     notification.onclick = () => {
-      if (!routerHistory) {
-        return;
-      }
-
-      routerHistory(`/transaction/${tx.tx_id}/`);
+      put(setNavigateTo(`/transaction/${ tx.tx_id }/`))
```

Comment:
Does it work without `yield`?

Reply:
It does because this is actually executed in a callback for a `onClick` event, not part of the `handleNewTx()` generator function.

It only benefits from being in the same scope as the generator to be able to use the `put` function.